src/components/shared/Subscribe.vue

```diff
@@ -44,19 +44,21 @@ export default {
     ...mapActions(['subscribe']),
     handleSubscribe() {
       this.frequency = this.frequencyModel.value;
-
       const { frequency, name, email } = this;
 
       this.subscribe({ frequency, name, email })
         .then(() => {
           this.showDialog('Email listesine kaydınız gerçekleştirildi.');
+          this.close();
         })
         .catch((e) => {
           const details = Object.values(e.response.data.errors || []).map(r => `<li>${r[0]}</li>`);
-
           this.showDialog('Hata: Kaydınız gerçekleştirilemedi.', `<ul>${details || ''}</ul>`);
         });
     },
+    handleEnter(){
```

Comment:
Eslint error is here: "Missing space before opening brace"

You can replace it with: `handleEnter() {`

Reply:
Yes, I really do. 
There is a problem in my setup. When I run `yarn lint`, it changes the line endings in all of the files, if I don't, I can't see the errors :(